envoy.dependency.check/envoy/dependency/check/abstract/release.py

```diff
@@ -22,10 +23,12 @@ def __init__(
             self,
             repo: github.AGithubRepo,
             version: str,
-            release: Optional[github.AGithubRelease] = None) -> None:
+            url: list,
```

Comment:
im thinking perhaps lets make this `asset_url` or similar something a bit more explicit